src/mpid/ch4/netmod/ofi/ofi_events.c

```diff
@@ -193,7 +193,7 @@ static int recv_event(struct fi_cq_tagged_entry *wc, MPIR_Request * rreq, int ev
                                             0 /* len */ ,
                                             MPIR_Comm_rank(c),
                                             MPIDI_OFI_comm_to_phys(c, r, vni_dst, vni_src),
-                                            ss_bits), tinjectdata, FALSE /* eagain */);
+                                            ss_bits), vni_dst, tinjectdata, FALSE /* eagain */);
```

Comment:
If we always think `vni_src` is local vni and `vni_dst` is remote vni, then here using `vni_dst` to progress ofi cqread is confusing with below `ofi_send` in this commit, which is using `vni_src` to progress.

I mean, this kind of change should be consistent, no matter send or recv, they both have local vni (endpoint) to send out data and remote vni (remote address) to deliver data.

Swapping order of src and dst in different function makes people hard to catch.

Reply:
Maybe using the names `vci_local` and `vci_remote` helps? They would have different values (`vci_src` or `vci-dst`) depending on whether it is receiver or sender. This could be more natural to understand.

Reply:
In terms of a single pt2pt message, there is a source, and there is a destination, sending vci and recving vci need be consistent for both sender and receiver, so a consistent naming is important -- i.e. where sender and receiver use same name to refer to source and dest.

In terms of address lookup, it is looking up remote address from a local vni, which means differently for sender and receiver.

Does this explains better? I believe it will raise more confusion if we mix the two.

Reply:
> In terms of a single pt2pt message, there is a source, and there is a destination, sending vci and recving vci need be consistent for both sender and receiver, so a consistent naming is important -- i.e. where sender and receiver use same name to refer to source and dest.

Sending vci and receiving vci is all depend on how you calculate it, with MPI_Send(dst, tag) and MPI_Recv(src, tag) and own rank.

Consistent calculation on both sender and receiver can guarantee same vci idx is used on two side. @sagarth can share with you our design for calculating idx.

For address lookup, remote delivery address should always be checked by addr[vci_local][vci_remote], means using vci_local `th` endpoint to deliver to `vci_remote` addr on the map.

I think the major conflict is that, we think which endpoint used for sending or receiving message should always be decided by local vci idx.

Reply:
Discussed with @shawnccx and I believe the comment is about using `vci_local/remote` in `fi_tsend/fi_tinject/fi_trecv` and I think it makes sense. I'll check and insert extra variable assignment with right names where necessary to make the code clearer.